packages/mdc-select/index.js

```diff
@@ -112,6 +112,8 @@ export class MDCSelect extends MDCComponent {
       deregisterMenuInteractionHandler: (type, handler) => this.menu_.unlisten(type, handler),
       notifyChange: () => this.emit(MDCSelectFoundation.strings.CHANGE_EVENT, this),
       getWindowInnerHeight: () => window.innerHeight,
+      addBodyClass: (className) => document.body.classList.add(className),
```

Comment:
why do you add this class to the document body? Why can't this class be added directly to the this.root_ element?

Reply:
He is looking at the same implementation for the drawer and doing the same here :smiley:

Reply:
thanks for the context @touficbatache